pkg/runs/testSelection.go

```diff
@@ -304,7 +302,8 @@ func selectTestsByTest(testCatalog launcher.TestCatalog, testSelection *TestSele
 	return err
 }
 
-func selectTestsByClass(testCatalog launcher.TestCatalog, testSelection *TestSelection, flags *utils.TestSelectionFlagValues) error {
+func selectTestsByClass(testCatalog launcher.TestCatalog, // Shouldn't this be used ?
```

Comment:
This is being used on line 188 of this file

Reply:
I was talking about the parameter, not the method.